--- DVDRipOMatic-0.95/DVDScan.orig	2006-10-07 13:26:12.000000000 +0100
+++ DVDRipOMatic-0.95/DVDScan	2007-03-11 22:33:42.000000000 +0000
@@ -397,6 +397,11 @@
  
  NumAudioStreams=$(cat "$MPlayerOutput" | grep "\[open\] audio stream" | wc -l)
 
+ ## Simple Patch here by dribble to support (hopefully) older and newer mplayer
+ if [ $NumAudioStreams -eq 0 ]; then
+  NumAudioStreams=$(cat "$MPlayerOutput" | grep "^audio stream:" | wc -l)
+ fi
+
  # Not all audio streams reported by MPlayer may actually exist.
  # Count which streams really exist.
 
@@ -412,6 +417,11 @@
 
    StreamDescription=`cat "$MPlayerOutput" | grep "\[open\] audio stream" | tail -n $[$NumAudioStreams - $Stream + 1] | head -n 1`
 
+   ## Simple Patch here by dribble to support (hopefully) older and newer mplayer
+   if [ -z "$StreamDescription" ]; then
+    StreamDescription=`cat "$MPlayerOutput" | grep "^audio stream:" | tail -n $[$NumAudioStreams - $Stream + 1] | head -n 1`
+   fi
+
    if [ -z "$StreamDescription" ]; then
      break;
    fi
@@ -619,4 +629,4 @@
 
 rm -f "$TCProbeOutput"
 rm -f "$MPlayerOutput"
-rm -f "$MPlayerAudioDetectOutput"
\ No newline at end of file
+rm -f "$MPlayerAudioDetectOutput"
